xtask/src/visualize_bundler_bench.rs

```diff
@@ -91,7 +91,12 @@ fn generate_scaling(
 
         // TODO: Avoid vector allocation -- not sure why this is necessary for
         // typechecker
-        let time_range = fitting_range(time_range_iter.collect::<Vec<f64>>().iter());
+        let time_range = {
+            let mut range = fitting_range(time_range_iter.collect::<Vec<f64>>().iter());
+            // Ensure the time range starts at 0 instead of the minimum time value.
+            range.start = 0.0;
+            range
```

Comment:
Is this just a `iter().max()` now?

Reply:
Yes, that's even better. I'll make that change.